### PR TITLE
Hugo: Add quote shortcode

### PR DESCRIPTION
**Pull request by anne-usmedia**
- Styling regular markdown quote
- Added quote shortcode
- Styling quote shortcode
- Added 300 weight font
- Added leading zeros in base style
- Changed lightest grey
- Added quote shortcode example page

For https://linear.app/usmedia/issue/CUE-122

### CODE REVIEW

File: hugo/content/en/examples/shortcodes/quote/index.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: Quote
+weight: 28
+---
+
+Next to the regular (very basic) [markdown blockquotes](/examples/basic/block#blockquotes), there is an option to use the `{{</* quote */>}}` shortcode. With this shortcode you can also add an author name and their role. But unlike the markdown blockquotes, the `quote` shortcode does not accept markdown.
```
Comment (JorindeUsMedia) [inline hugo/content/en/examples/shortcodes/quote/index.md:6]: Maybe also link to this page from the basic/block page quote section?  
"For advanced quote shortcode see: .."

File: hugo/assets/scss/base/base.scss [hunk 2]
```diff
@@ -12,18 +13,14 @@
...
+    --caption-color: #{ transparentize($c-grey--darkest, 0.3) };
     --list-color: #{ $c-blue };
-    --table-border-color: #{ transparentize($c-blue--light, .85) };
-    --table-header-border-color: #{ transparentize($c-blue--light, .85) };
+    --table-border-color: #{ transparentize($c-blue--light, 0.85) };
+    --table-header-border-color: #{ transparentize($c-blue--light, 0.85) };
     --pre-color: #{ $c-grey--darkest };
     --pre-background-color: #{ $c-blue--lighter };
     --pre-border-color: #{ $c-blue--lighter };
-    --quote-background-color: #{ $c-grey--lighter };
```
Comment (JorindeUsMedia) [inline hugo/assets/scss/base/base.scss:22]: why are these removed here? for base stuff it's best to set all variables at the top so it's easily to change them all if you have to (for instance different styling needed when base stuff is shown on a blue background)

**Review by JorindeUsMedia — CHANGES_REQUESTED** [review]

**Review by Lyoness — CHANGES_REQUESTED** [review]
Quote link needs to be added to hugo/content/en/examples/_index.html

I can see it here: https://deploy-preview-319--cue.netlify.app/examples/shortcodes/quote/

But not here: 
https://deploy-preview-319--cue.netlify.app/examples/

**Comment by Lyoness** [comment]
Ah yes, thanks @anne-usmedia.

**Review by Lyoness — APPROVED** [review]
Importing to Gerrit

**Comment by Lyoness** [comment]
https://review.gerrithub.io/c/cue-lang/cuelang.org/+/550790

**Comment by anne-usmedia** [comment]
@Lyoness just rebased this branch

**Comment by myitcv** [comment]
Thanks @anne-usmedia - reimporting.

**Pull request closed without merging**